DLaB.EarlyBoundGenerator/SpecifyEntities.cs

```diff
@@ -86,20 +87,13 @@ private void BtnSave_Click(object sender, EventArgs e)
 
         private void BtnRefresh_Click(object sender, EventArgs e)
         {
-            try
-            {
-                Enable(false);
+            Enable(false);
 
-                // Clear existing entity list
-                ((XrmToolboxCommon.PropertyInterface.IEntityMetadatas) CallingControl).EntityMetadatas = null;
+            // Clear existing entity list
+            ((XrmToolboxCommon.PropertyInterface.IEntityMetadatas) CallingControl).EntityMetadatas = null;
 
-                // Retrieve entities
-                RetrieveEntityMetadatasOnLoad(LoadEntities);
-            }
-            finally
-            {
-                Enable(true);
```

Comment:
I don't see it getting re-enabled... Am I missing something?

Reply:
It now just follows the same path as when the form is first loaded. So LoadEntities calls Enable(true) once it finished loading the entities.

Reply:
Since it follows the same path as initially loading the form, the LoadEntities method will re-enable everything once it's done.